pkgs/os-specific/linux/pam/default.nix

```diff
@@ -37,11 +37,13 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   preConfigure = lib.optionalString (stdenv.hostPlatform.libc == "musl") ''
```

Comment:
This appears to be no longer necessary: https://github.com/linux-pam/linux-pam/commit/1d1d3b59af50c40efcc74f9c3f37b446a921e83e

`pkgsMusl.pam` builds without it.

Reply:
updated this patch to remove the preConfigure.

Alpine (where we took this from) also dropped these sed bits <https://gitlab.alpinelinux.org/alpine/aports/-/commit/e51f5c2997fd7a2140d417b6ddad1cec2cec9952>